index.js

```diff
@@ -286,8 +286,18 @@ async function init() {
   }
   if (needsTypeScript) {
     render('config/typescript')
+
+    // Render tsconfigs
+    render('tsconfig/base')
+    if (needsCypress) {
+      render('tsconfig/cypress')
+    }
+    if (needsVitest) {
```

Comment:
Only the Vitest template needs an additional tsconfig, because it runs in Node.js with JSDOM, which is different than other files under `src/`.

Cypress Component Testing runs in an actual browser, and the Cypress type definitions will be brought in by `import { mount } from '@cypress/vue'`, so there's no need for a standalone tsconfig.